pages/Configuring/Animations.md

```diff
@@ -1,4 +1,7 @@
+# Animations
```

Comment:
We don't add titles in the beginning, since Hugo already puts the name of the page as the title. So having it here as well will put 2 titles and looks bad.

Reply:
That's actually good to know. Thanks.

We can blame vaxry for not giving me a copy of the backend so that I could test, would've been eleminated indev otherwise. Regardless, will fix later today.